src/components/CardCompanySection/CardCompanySection.tsx

```diff
@@ -0,0 +1,20 @@
+import { CARD_COMPANY } from '../../constants';
+import Dropdown from '../Dropdown/Dropdown';
+import { InputSection } from '../InputSection/InputSection';
+
+type CardCompanySection = {
+  company: string;
+  handleCompanySelect: (value: string) => void;
```

Comment:
```suggestion
  handleCompanySelect: (value: CardCompany) => void;
```
`CardCompanySection`는 도메인 관련 컴포넌트이기 때문에 `handleCompanySelect` 함수의 매개변수 `value` 타입을 조금 더 좁히는게 안전할 것 같아요🤔